tutorials/xr/openxr_settings.rst

```diff
@@ -278,6 +290,13 @@ the user is holding a controller.
 If only a hand interaction profile is supplied any runtime should use hand
 interaction even if a controller is being held.
 
+Spatial Entity
+~~~~~~~~~~~~~~
```

Comment:
```suggestion
Spatial Entities
~~~~~~~~~~~~~~~~
```
How it's usually called

Reply:
But it's inconsistent with other places, the documentation uses "spatial entities", the settings also says "render model" but the entry here and the associated page says "render models", it's weird to me that the section "Spatial Entity" links to the page "spatial entities", also the OpenXR extension name is "spatial entities" not "spatial entity"